app/javascript/components/home/ExceedYourProfessionalGoalsSection.jsx

```diff
@@ -2,12 +2,12 @@ import React from 'react';
 
 const ExceedYourProfessionalGoalsSection = () => {
     return (
-        <>
+        <p>
```

Comment:
I'm changing these to `p` tags just to make sure they're semantically meaningful :)